### PR TITLE
Add create_user command with api key support

### PR DESCRIPTION
**Pull request by sevein**
Simple command to allow us create users from the command line with predefined parameters like email, password or API key. This is useful in fully automated deployments - I'm planning to use it in rdss-archivematica.

Related: https://github.com/JiscRDSS/archivematica/pull/4.

### CODE REVIEW

File: storage_service/common/management/commands/create_user.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        if user is None:
+            raise CommandError('User not found!')
+
+        user.email = options['email']
+        user.set_password(options['password'])
+        user.save()
+        print('User updated.')
+
+        try:
```
Comment (helenst) [inline storage_service/common/management/commands/create_user.py]: I think `create_or_update` could be used here.

Comment (sevein) [reply storage_service/common/management/commands/create_user.py]: Nice, I never remember the convenience methods!
How would the code look like if I want to integrate it with `create_superuser`?

Comment (helenst) [reply storage_service/common/management/commands/create_user.py]: I think that would be (I've not tested this)
```
ApiKey.objects.create_or_update(user=user, defaults={'key': options['api_key']})
```

and if you want to know whether it was created or updated:
```
obj, created = ApiKey.objects.create_or_update(user=user, defaults={'key': options['api_key']})
if created:
    print('API key created')
```

etc.

**Review by lower29 — APPROVED** [review]
My django isn't as good as Helen's, I wouldn't have spotted that!

I don't have any further concerns so address hers and I think we're good. 👍

File: storage_service/common/management/commands/create_user.py (hunk 1 above)
Comment (sevein) [reply storage_service/common/management/commands/create_user.py]: Thanks, it worked! :-)

**Pull request merged**